index.js

```diff
@@ -156,8 +156,54 @@ var znFilterMatcher = (function() {
 	};
 
 	/**
+	 * Is var an object?
+	 *
+	 * @param	x
+	 * @returns	{Boolean}
+	 * @author	David McNelis <david.mcnelis@wizehive.com>
+	 * @since	1.1.0
+	 */
+	function isObject(x) {
+		return (!!x) && (x.constructor === Object);
+	};
+
+	/**
+	 * Is var an empty object?
+	 *
+	 * @param	x
+	 * @returns	{Boolean}
+	 * @author	David McNelis <david.mcnelis@wizehive.com>
+	 * @since	1.1.0
+	 */
+	function noKeys(x) {
```

Comment:
@dwmcnelis I'd use a helper to determine whether it's an object....

Reply:
Done